chartboost-mediation-android-sdk-demo/compose/ChartboostMediationDemoCompose/build.gradle.kts

```diff
@@ -0,0 +1,91 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "com.chartboost.mediation.sdk.demo"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "com.chartboost.mediation.sdk.demo"
+        minSdk = 21
+        targetSdk = 34
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary = true
+        }
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro",
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_17
+        targetCompatibility = JavaVersion.VERSION_17
+    }
+    kotlinOptions {
+        jvmTarget = "17"
+    }
+    buildFeatures {
+        compose = true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.5.7"
+    }
+    packaging {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+}
+
+dependencies {
+    // Chartboost Mediation SDK
+    implementation("com.chartboost:chartboost-mediation-sdk:4.9.0")
+
+    // Chartboost Mediation adapters
+    implementation("com.chartboost:chartboost-mediation-adapter-admob:4.+")
```

Comment:
update to 5.+?

Reply:
The update work to will be done separately by @bwised.